pkg/controller/jobs/job/job_controller_test.go

```diff
@@ -692,7 +727,7 @@ func TestReconciler(t *testing.T) {
 						Type:    kueue.WorkloadFinished,
 						Status:  metav1.ConditionTrue,
 						Reason:  "FailedToStart",
-						Message: `invalid admission check PodSetUpdate: conflict for nodeSelector: conflict for key=provisioning, value1=on-demand, value2=spot`,
+						Message: `invalid admission check PodSetUpdate: conflict for nodeSelector: conflict for key=provisioning, value1=spot, value2=on-demand`,
```

Comment:
Why do we need this change?

Reply:
I've reversed the Merge direction in `func Merge(meta *metav1.ObjectMeta, spec *corev1.PodSpec, info PodSetInfo) error ` (no especially for this test) , now the existing value "spot" is named "value1", it makes better sence.

Reply:
Ah, it makes sense. I confirmed the function again.